src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -598,38 +598,56 @@ void destroyAllOntimeIntervalThreads(std::string tenant)
 
```

Comment:
It is remarkable that case of 674 and 675, as they are not related with CPrs and they are failing (indeed, they seem to be related with subscriptions).

I will look into them before merging this PR.